app/restore_to.go

```diff
@@ -0,0 +1,145 @@
+package app
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/rancher/longhorn-engine/replica"
+	"github.com/rancher/longhorn-engine/util"
+	"github.com/urfave/cli"
+)
+
+const DefaultImageFormat = "qcow2"
+
+var SupportedImageFormats = []string{
+	"rbd",
```

Comment:
It would be a nightmare to test again so many formats.

I say we only support `qcow2`, and `raw` for now. We can expand the list when there are use cases.